src/src/components/App/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
Why is src/public/index.html getting deleted? Sorry didn't know where else to comment this.

Reply:
I see, makes sense.